packages/trpc/server/routers/viewer/saml.tsx

```diff
@@ -30,7 +30,7 @@ export const samlRouter = router({
       }
 
       // Retrieve the SP SAML Config
-      const SPConfig = samlSPConfig.get();
+      const SPConfig = await samlSPConfig.get();
```

Comment:
https://github.com/boxyhq/jackson/pull/667#issuecomment-1322630427

Reply:
Thanks @emrysal, added the upgrade note here - https://boxyhq.com/docs/jackson/upgrade#upgrading-to-v136

Reply:
Loving the context for the change! 👍